src/java.compiler/share/classes/javax/lang/model/SourceVersion.java

```diff
@@ -496,8 +496,7 @@ public static SourceVersion latestSupported() {
      * followed only by characters for which {@link
      * Character#isJavaIdentifierPart(int)} returns {@code true}.
      * This pattern matches regular identifiers, keywords, contextual
-     * keywords, and the literals {@code "true"},
-     * {@code "false"}, {@code "null"}.
+     * keywords, boolean literals, and the null literal.
```

Comment:
I don't think it is necessarily more clear remove the explicit listing of the strings in question.

Reply:
Perhaps. I just thought that it's strange to spell them out only here, but refer to them through "x literal(s)" everywhere else in that file.